src/include/migraphx/op/gather.hpp

```diff
@@ -17,35 +18,33 @@ namespace op {
 
 struct gather
 {
-    int axis = 0;
+    int64_t axis = 0;
 
     template <class Self, class F>
     static auto reflect(Self& self, F f)
     {
         return pack(f(self.axis, "axis"));
     }
 
+    value attributes() const
+    {
+        value attr;
```

Comment:
This is only for the normalize_axes is attribute. I would call it `normalize` or `normalize_axes` to make it a little bit clearer.

Reply:
OK, will change that.

Reply:
changed

Reply:
I meant rename the `attr` variable to `normalize` not the name of the attribute.

Reply:
fixed back